.github/workflows/pypi_packaging.yml

```diff
@@ -0,0 +1,32 @@
+name: Publish to PyPi
+
+# Controls when the action will run.
+# Triggers the workflow on push or pull request events
+on: [push]
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    # The type of runner that the job will run on
+    runs-on: ubuntu-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Setup Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.10'
+
+      - name: Setup Env
+        run: |
+          python3 -m pip install --upgrade pip twine build
+
+      - name: Build
+        run: python3 -m build
+
+      - name: Publish to PyPi
+        run: python3 -m twine upload --username __token__ --password ${{ secrets.PYPI_TOKEN }} --repository testpypi dist/*
```

Comment:
`testpypi` needs be changed to `pypi` before merging to `cromshell2` branch. Leaving as is for now for testing

Reply:
leaving testpypi in for draft releases